packages/webapp/src/components/app/AuthorizedRoutes.tsx

```diff
@@ -27,34 +27,36 @@ const AddService = lazy(
 const EditService = lazy(
 	() => /* webpackChunkName: "EditServicePage" */ import('~pages/services/editService')
 )
-const ServiceKiosk = lazy(
-	() => /* webpackChunkName: "ServiceKioskPage" */ import('~pages/services/serviceKiosk')
+const ServiceEntry = lazy(
+	() => /* webpackChunkName: "ServiceEntryPage" */ import('~pages/services/serviceEntry')
 )
 
 export const AuthorizedRoutes: FC = memo(function AuthorizedRoutes() {
 	return (
-		<>
-			<ContainerLayout>
-				<PushNotifications />
-				<Suspense fallback={<LoadingPlaceholder />}>
-					<Switch>
-						<Route exact path={ApplicationRoute.Index} component={Index} />
-						<Route path={ApplicationRoute.Account} component={Account} />
-						<Route path={ApplicationRoute.Clients} component={Clients} />
-						<Route path={ApplicationRoute.Specialist} component={Specialist} />
-						<Route path={ApplicationRoute.Reporting} component={Reporting} />
-						<Route path={ApplicationRoute.Tags} component={Tags} />
-						<Route exact path={ApplicationRoute.Services} component={ServicesIndex} />
-						<Route path={ApplicationRoute.AddService} component={AddService} />
-						<Route path={ApplicationRoute.EditService} component={EditService} />
-						<Route path={ApplicationRoute.ServiceKiosk} component={ServiceKiosk} />
-
-						{/* Slash path matches all. It's used as a catch-all here for not-found routes */}
-						<Route path={ApplicationRoute.Index} component={NotFound} />
-					</Switch>
-				</Suspense>
-			</ContainerLayout>
-			<Footer />
-		</>
+		<Switch>
+			<Route path={ApplicationRoute.ServiceKioskMode} component={ServiceEntry} />
+			<>
+				<ContainerLayout>
+					<PushNotifications />
+					<Suspense fallback={<LoadingPlaceholder />}>
+						<Switch>
+							<Route exact path={ApplicationRoute.Index} component={Index} />
+							<Route path={ApplicationRoute.Account} component={Account} />
+							<Route path={ApplicationRoute.Clients} component={Clients} />
+							<Route path={ApplicationRoute.Specialist} component={Specialist} />
+							<Route path={ApplicationRoute.Reporting} component={Reporting} />
+							<Route path={ApplicationRoute.Tags} component={Tags} />
+							<Route exact path={ApplicationRoute.Services} component={ServicesIndex} />
+							<Route path={ApplicationRoute.AddService} component={AddService} />
+							<Route path={ApplicationRoute.EditService} component={EditService} />
+							<Route path={ApplicationRoute.ServiceEntry} component={ServiceEntry} />
+							<Route path={ApplicationRoute.Index} component={NotFound} />
+						</Switch>
+					</Suspense>
+				</ContainerLayout>
+				<Footer />
+				{/* Slash path matches all. It's used as a catch-all here for not-found routes */}
```

Comment:
That comment should go above line 53 `<Route path={ApplicationRoute.Index} component={NotFound} />`

Reply:
Another good catch.